ci-operator/step-registry/baremetalds/arbiter/e2e/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
+  - edge-enablement-approvers
+  - jogeo
```

Comment:
Please also add dhensel-rh to all OWNERS files.

Reply:
Done, added.